reqs/requirements-dev-lite.txt

```diff
@@ -13,4 +13,4 @@ pytest-timeout>=2.1.0
 requirements-parser>=0.2.0
 s3fs>=0.5.1
 snapshottest==0.6.0 # GE Cloud atomic renderer tests
-sqlalchemy>=1.3.18,<2.0.0
+sqlalchemy>=1.3.18,<1.4.42
```

Comment:
`_rfc_1738_quote()` method is needed for the `snowflake` and `trino` pacakges. 
* [snowflake has already addressed the issue by adding the method to their codebase](https://github.com/snowflakedb/snowflake-sqlalchemy/commit/abf7edb09d02cc0f5297329a9dfd93c8e6a9b576)
* pinning sqlalchemy as `1.4.42` until [trino-python-client](https://github.com/trinodb/trino-python-client) is released that includes the following change : [following change](https://github.com/trinodb/trino-python-client/blob/master/trino/sqlalchemy/util.py#L9-L10) which should be`0.319.0`